docs/_pages/fr/effectuer-des-demandes.md

```diff
@@ -131,10 +130,62 @@ GET /forms/{formID}/submission/{submissionName}
+
+#### Pour récupérer les fichiers téléchargés vers un formulaire en toute sécurité :
+
+Vous recevrez chaque soumission sous forme de fichier JSON brut comprenant des liens directs vers les fichiers, s'ils sont joints. Pour des raisons de sécurité, ces liens de téléchargement direct ne sont valables que pendant 10 secondes.
+
+Les fichiers étant analysés lors de leur soumission, ils peuvent également être signalés comme potentiellement malveillants, en fonction du logiciel d'analyse des fichiers. Vous devez décider quoi faire avec les fichiers signalés — soit que vous les téléchargez ou non, s'ils comportent un attribut de métadonnées « malveillant » ou « mauvais ».
+
+Ajoutez du code pour gérer les cas où les fichiers sont signalés et <code>isPotentiallyMalicious:true</code> est rencontré. Vous pourrez ensuite accéder aux données soumises et aux fichiers sécurisés joints dans votre système. 
+
 ### **Confirmer** des soumissions de formulaires
 
 L’étape de confirmation permet de s’assurer que les formulaires soumis sont exploitables et s’affichent comme prévu avant qu’ils ne soient définitivement retirés de la base de données du système Formulaires GC. 
 
+Avant de confirmer une réponse, assurez-vous d'avoir accès à tous les fichiers joints, le cas échéant. Ces liens de téléchargement de fichiers ne sont disponibles que temporairement, car ils ne sont valables que pendant 10 secondes. Si vous n'avez pas reçu tous les fichiers, téléchargez à nouveau le même formulaire soumis pour obtenir de nouveaux liens de téléchargement. Signalez tout problème lié aux fichiers ou aux données. Une fois que vous êtes certain d'avoir bien reçu toutes les données et tous les fichiers soumis, confirmez que toutes les données ont été transférées avec succès. Les réponses et les fichiers seront ensuite supprimés de GC Forms après 30 jours.
+
+#### Exemple de données de réponses avec fichiers
+
+Les données ressembleront à ceci lorsque des fichiers sont joints :
+
+<code>
+{
+
+   "createdAt":1749476854628,
+
+   "status":"New",
+
+   "confirmationCode":"714dfe46-6fa1-4281-8d15-a39bcebc3c4f",
+
+   "answers":"{\"1\":\"Test1\",\"2\":\"form_attachments/2025-06-09/8b42aafd-09e9-44ad-9208-d3891a7858df/output.txt\",\"3\":\"form_attachments/2025-06-09/9064b3c7-eee5-4599-99c8-a257b2b5f37d/a0393b10-396c-4b8d-a97c-15394fddda86.jpg\",\"4\":\"form_attachments/2025-06-09/0c7c3414-05e2-4ae6-a825-683857e4c0c4/IMG_0441.jpeg\"}",
+
+   "checksum":"cc33cb49f6c088bf98b7315794db216e",
+
+   "attachments":[
+
+      {
+
+         "name":"output.txt",
+
+         "directLinkToFile":"((s3 link))",
+
+         "isPotentiallyMalicious":true
+
+      }
+
+   ]
```

Comment:
Same thing here

With the current version of the code + what we will be merging soon [MD5 for attachments](https://github.com/cds-snc/forms-api/pull/249), the JSON for the list of attachments will be

```JSON
"attachments":[
  {
    "id": "04d8aff7-25d7-49e5-8f01-77a8b6fba214",
    "name":"output.txt",
    "downloadLink":"https://???",
    "isPotentiallyMalicious":true,
    "md5": "54b0c58c7ce9f2a8b551351102ee0938"
  }
]
```